chain/store/store.go

```diff
@@ -880,26 +922,29 @@ func (cs *ChainStore) GetRandomness(ctx context.Context, blks []cid.Cid, round i
 		mtb := nts.MinTicketBlock()
 
 		if int64(nts.Height()) <= round {
```

Comment:
try:
` 		if int64(nts.Height()) <= round  || mtb.Height == 0`
instead. And remove the below special case.